docs/reference/dropbox.md

```diff
@@ -0,0 +1,118 @@
+# Dropbox Connector
+
+The [Elastic Dropbox connector](../connectors/sources/dropbox.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+## Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+## Usage
+
+To use this connector as a **connector client**, use the **Dropbox** tile from the connectors list or **Customized connector** workflow.
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+Creating a Dropbox application is required to help the Dropbox connector to authenticate.
+
+## Dropbox API Authorization
+
+### Create Dropbox OAuth App
+
+To generate App Key and App Secret, follow the next steps:
+1. Register a new app in the [App Console](https://www.dropbox.com/developers/apps) and select Full Dropbox API app and choose the following required permissions.
+    - files.content.read
+    - sharing.read
+
+2. Once the app is created, please store **app key** and **app secret** for later configuration use.
+
+### Generate a Refresh Token
+
+To generate a refresh token, follow the next steps:
+1. Hit the following URL with the generated APP_KEY on your browser:
+    ```shell
+    https://www.dropbox.com/oauth2/authorize?client_id=<APP_KEY>&response_type=code&token_access_type=offline
+    ```
+    
+    In the HTTP response, you will get an **authorization code** that will be used to generate a refresh token.
+
+    **Note:** Authorization code can only be used once to create a refresh token.
+
+2. Perform the following POST API call in the terminal:
+    ```shell
+    curl -X POST "https://api.dropboxapi.com/oauth2/token?code=<AUTHORIZATION_CODE>&grant_type=authorization_code" -u "<APP_KEY>:<APP_SECRET>"
+    ```
+    Store the refresh token from the response and use it in the connector configuration.
+    **Note:** Make sure the response has a similar list of following scopes:
```

Comment:
It seems like it can be helpful to explain why the refresh token requires 4 scoped permissions, more than the two required to initially create the OAuth app (files.content.read and sharing.read).

Reply:
`account_info.read` is the default access and once you set `files.content.read` permission, it'll automatically enable `files.metadata.read` access. So, there will be total 4 resultant permissions present in the cURL response. So, i think that's fine to not add this in the doc.